sections/footer.liquid

```diff
@@ -73,14 +73,36 @@
                       {%- endfor -%}
                     </ul>
                   {%- endif -%}
+                {%- when 'brand_information' -%}
+                  <div class="footer-block-brand-information">
+                    {%- if settings.brand_image != blank -%}
+                      {%- assign image_size_2x = settings.brand_image.width | times: 2 | at_most: 5760 -%}
+                      <img
+                        srcset= "{{ settings.brand_image | image_url: width: settings.brand_image.width }}, {{ settings.brand_image | image_url: width: image_size_2x }} 2x"
+                        src="{{ settings.brand_image | image_url: width: 760 }}"
+                        alt="{{ settings.brand_image.alt | escape }}"
+                        class="global-media-settings"
+                        loading="lazy"
+                        width="{{ settings.brand_image.width }}"
+                        height="{{ settings.brand_image.height }}"
+                        style="max-width: min(100%, {{ settings.brand_image_width }}px);"
+                      >
+                    {%- endif -%}
+                    <h2 class="footer-block__heading">{{ settings.brand_headline }}</h2>
+                    <p>{{ settings.brand_description }}</p>
```

Comment:
We ok with rendering those elements when not content is added in the settings for them ?

Reply:
Updated to add a check for emptiness.